src/components/utils/Spacing.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'react-emotion'
+
+export enum Size {
+  SM,
+  MD,
+  LG,
+}
+
+const spaceSizes: Record<Size, number> = {
+  [Size.SM]: 12,
+  [Size.MD]: 16,
+  [Size.LG]: 20,
+}
+
+type spacingAttribute =
+  | 'marginTop'
+  | 'marginBottom'
+  | 'marginRight'
+  | 'marginLeft'
+  | 'paddingTop'
+  | 'paddingBottom'
+  | 'paddingRight'
+  | 'paddingLeft'
+
+type Props = Partial<Record<spacingAttribute, Size>>
+
+export const Spacing = styled('div')((props: Props) =>
```

Comment:
I love `<Spacing />`! So much so that we already have it (Sam introduced it earlier in app and I copied it to web). We should use a standardized one though I think, so lets try to unify the `<Spacing />`s 👍

Reply:
åhå nice! Maybe for another time though? 😁

Reply:
Yes, pure focus now 🚀